src/components/cc-tile-requests/cc-tile-requests.types.d.ts

```diff
@@ -1,5 +1,25 @@
-type RequestsData = [
+export type RequestsData = [
   number, // Start timestamp in milliseconds. Expected to be rounded to the hour of its respective TZ.
   number, // End timestamp in milliseconds. Expected to be rounded to the hour of its respective TZ.
   number, // Number of request during this time window.
 ]
+
+export type TileRequestsState = TileRequestsStateLoaded | TileRequestsStateLoading | TileRequestsStateError | TileRequestsStateEmpty;
+
+export interface TileRequestsStateLoaded {
+  type: 'loaded';
+  data: Array<RequestsData>;
+}
+
+export interface TileRequestsStateLoading  {
+  type: 'loading';
+}
+
+export interface TileRequestsStateError {
+  type: 'error';
+}
+
+export interface TileRequestsStateEmpty {
```

Comment:
question: do we really need a separate state for emptiness? Couldn't we use the 'loaded' state and check that the data is empty?

question: what happens if the state is loaded and data is an empty array? Is it the same rendering than when state is 'empty'?

Reply:
This is a good question :thinking: 

Initially, we went for an `empty` state because that's what the `cc-tile-metrics` does and it's the most recent of the tile components (in fact it already had a state).

We figured we would discuss this point in sync but forgot to mention it within the discussion.

Anyway, I think you have a valid point but I'd also like @Galimede to weigh in since he built the `cc-tile-metrics` and I remember there were some hesitations about how to handle empty data :thinking: 

I would prefer we reach a consensus and apply the same logic to both components even though the API is not the same: it would be great if our UI components had the same logic and the smart would adapt to it if necessary.